pkg/controller/pxc/replication.go

```diff
@@ -212,7 +213,7 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileReplication(cr *api.PerconaXtra
 		if err != nil {
 			return errors.Wrap(err, "add label to main replica pod")
 		}
-		r.logger(cr.Name, cr.Namespace).Info("Replication pod has changed", "new replication pod", primaryPod.Name)
+		logf.FromContext(ctx).Info("Replication pod has changed", "new replication pod", primaryPod.Name)
```

Comment:
why not just `log`?

Reply:
No reason, I missed that we have it declared.